packages/backend/services/stream-indexer/index.js

```diff
@@ -0,0 +1,167 @@
+require("dotenv").config();
+const ethers = require("ethers");
+const db = require("../db/db");
+
+const SIMPLE_STREAM_ABI = [
```

Comment:
Maybe we can export this from a separate file to avoid polluting this one.

Reply:
Moved to `backend/abi/SimpleStream.json`.